### DISCUSSION TITLE
Persistent record docs modified at the Journal day

### DISCUSSION THREAD

**Discussion by vankeoa** [toeverything/AFFiNE#6184] [Ideas]
### Description

journals中的updated如实的关联docs中实际调整的是否更好，可以通过updated了解到对应日期的工作调整。

### Use case

！[上传截屏2024-03-19 07.56.11[07.56.11.p]
![Uploading 截屏2024-03-19 07.56.11.png…]()

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by pengx17** [comment]
我没太能理解你的意思，你是说假如一篇文章 "foo" 在日期 A/B/C 都有编辑过，那么在日期 A/B/C 的 journal 里都显示 foo 吗？另外图片挂了。

**Reply by vankeoa** [reply]
是的，目前是全部展示，并不知道当天做了那些事情！不是整体体现体现的只是当天处理的，因为有的笔记不是一天完成的。

**Reply by vankeoa** [reply]
！[上传截屏.png...]()

**Reply by vankeoa** [reply]
！[上传截屏.png...]()不知道为什么我的上传总是有问题

**Reply by pengx17** [reply]
ok，没关系。我理解你的需求。这个可能得让产品同学看看 @hwangdev97

**Reply by hwangdev97** [reply]
We have also discussed this internally, and I will document this requirement to address it in future improvements.
